.github/workflows/deploy-prod.yml

```diff
@@ -1,45 +1,54 @@
 name: deploy-prod
 
 on:
-  workflow_dispatch:
-    inputs:
-      semver-type:
-        type: choice
-        description: The semver type
-        default: minor
-        options:
-        - major
-        - minor
-        - patch
+    workflow_dispatch:
+        inputs:
+            semver-type:
+                type: choice
+                description: The semver type
+                default: minor
+                options:
+                    - major
+                    - minor
+                    - patch
+            restart-type:
+                type: choice
+                description: Soft or hard restart
+                default: soft
+                options:
+                    - soft
+                    - hard
 jobs:
-  snap:
-    runs-on: ubuntu-latest
-    timeout-minutes: 10
-    steps:
-      - name: Checkout KMQ_Discord
-        uses: actions/checkout@v2
+    snap:
+        runs-on: ubuntu-latest
+        timeout-minutes: 10
+        steps:
+            - name: Checkout KMQ_Discord
+              uses: actions/checkout@v2
+              with:
+                  fetch-depth: 0
+                  token: ${{ secrets.PAT }}
```

Comment:
I think I missed where secrets.PAT is declared

Reply:
This wasn't added in this commit, it's in the repo's secret key vault